core-bundle/src/Controller/AbstractFragmentController.php

```diff
@@ -114,11 +114,16 @@ protected function addHeadlineToTemplate(Template $template, $headline): void
     /**
      * @param string|array $cssID
      */
-    protected function addCssAttributesToTemplate(Template $template, string $templateName, $cssID, array $classes = null): void
+    protected function addCssAttributesToTemplate(Template $template, string $templateName, $cssID, array $classes = null, string $overrideCssID = null): void
     {
         $data = StringUtil::deserialize($cssID, true);
         $template->class = trim($templateName.' '.($data[1] ?? ''));
-        $template->cssID = !empty($data[0]) ? ' id="'.$data[0].'"' : '';
+
+        if (!empty($overrideCssID)) {
```

Comment:
```suggestion
        if (null !== $overrideCssID) {
```
shouldn’t this be a null check so that I can overwrite the CssID with an empty string?

Reply:
I don't think that's intended. We're passing the CSS ID from an alias element to the included element, only if the value is not empty there.